src/game.js

```diff
@@ -14,7 +15,15 @@ class Game {
 		game.token = token
 		dbGame.push(game);
 		game.session = `http://localhost:3000/game?token=${token}`;
-
+		GameBd.sync()
+		  .then(() => GameBd.create({
+		    gameId: game.id,
+		    token: game.token,
+		    session: game.session
+		  }))
+		  .then(game => {
+		    console.log(JSON.stringify(game));
+		  });
```

Comment:
refactor this to return the right promise with the date of the newly created game